src/macros.cr

```diff
@@ -61,7 +61,17 @@
 # p.copy_with x: 3   # => #<Point(@x=3, @y=2)>
 # p                  # => #<Point(@x=0, @y=2)>
 # ```
-macro record(name, *properties)
+macro record(name, *properties, **kwargs)
+  {% raise "#{name} does not accept named arguments
```

Comment:
Referring to `name` doesn't make sense here. It's about the macro `record`.
```suggestion
  {% raise "Macro `record` does not accept named arguments.
```

Reply:
named _parameters_